tests/phpunit/tests/image/editorImagick.php

```diff
@@ -641,4 +641,53 @@ public function test_directory_creation() {
 
 		$this->assertNotWPError( $saved );
 	}
+
+	/**
+	 * Tests that the alpha channel of PDFs is removed from PDF previews.
+	 *
+	 * Only affects systems with Ghostscript version >= 9.14.
+	 *
+	 * @ticket 39216
+	 *
+	 * @covers WP_Image_Editor_Imagick::remove_pdf_alpha_channel
+	 */
+	public function test_remove_pdf_alpha_channel_should_remove_the_alpha_chanel_in_preview() {
```

Comment:
```suggestion
	public function test_remove_pdf_alpha_channel_should_remove_the_alpha_channel_in_preview() {
```
Typo: Never remove the chanel, it makes things smell nice.